deploy/02_clusterrole.yaml

```diff
@@ -0,0 +1,58 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: secondary-scheduler-operator
+rules:
+- apiGroups:
+  - operator.openshift.io
+  resources:
+  - "*"
+  verbs:
+  - "*"
+- apiGroups:
+  - monitoring.coreos.com
+  resources:
+  - servicemonitors
+  verbs:
+  - "*"
+- apiGroups:
+  - secondaryschedulers.operator.openshift.io
+  resources:
+  - "*"
+  verbs:
+  - "*"
+- apiGroups:
+  - ""
+  resources:
+  - services
+  - pods
+  - configmaps
+  - secrets
+  - namespaces
+  - nodes
+  - pods/eviction
+  - events
+  verbs:
+  - "*"
+- apiGroups:
+  - rbac.authorization.k8s.io
+  resources:
+  - roles
+  - rolebindings
+  verbs:
+  - '*'
+- apiGroups:
+  - scheduling.k8s.io
+  resources:
+  - priorityclasses
```

Comment:
does this need this permission? Looks like a lot was copied from descheduler-operator, so we should double check these

Reply:
No, I do not think so. It was indeed copied and I will remove those.